homework01/artur/vector.h

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+
+class Vector
+{
+ public:
+  Vector();
+  Vector(const int, const int);
```

Comment:
Why `const int` ?

Reply:
No particular reason, shouldn't be there.